subsys/bluetooth/mesh/health_cli.c

```diff
@@ -461,8 +430,9 @@ int bt_mesh_health_fault_get(uint16_t addr, uint16_t app_idx, uint16_t cid,
 	};
 	int err;
 
-	err = cli_prepare(&param, OP_HEALTH_FAULT_STATUS);
+	err = cli_prepare(&param, OP_HEALTH_FAULT_STATUS, addr);
 	if (err) {
+		printk("fault get failed to prepare\n");
```

Comment:
There should be no printk usage in the stack itself. Only on the app side. If you want to keep this then change it to a BT_ERR and include the actual error value through `%d` in the message.

Reply:
Sorry, this sneaked in while I was debugging it. Will remove. Thanks for noticing this!